test/workflows/components/workflows.libsonnet

```diff
@@ -360,7 +360,15 @@
             ]),  // py lint
             $.parts(namespace, name, overrides).e2e(prow_env, bucket).buildTemplate("setup-cluster", testWorkerImage, [
               "/usr/local/bin/create-eks-cluster.sh",
-            ]),  // setup cluster
+            ],
+            # need to specify kubernetes version to be created in AWS EKS
+            env_vars=[
+              {
+                name: "EKS_CLUSTER_VERSION",
+                value: "1.19",
```

Comment:
@Jeffwan Can't we update it to a later version?

Reply:
Yes. EKS supports up to 1.22 at this moment. However, seems we are still using 1.19 libraries and it may brings compatibility issue?  

https://github.com/kubeflow/training-operator/blob/ce82602697d6d4e45d8ac88b50e0d22a6aa39b63/go.mod#L14-L17
I would suggest to bump the version once we upgrade libraries.

Reply:
@johnugeorge @Jeffwan 
If we want to use K8s v1.22, we should upgrade `eksctl` version in  AWS CI worker image.

Ref:
  - https://github.com/kubeflow/katib/pull/1758#issuecomment-988147449
  - https://github.com/kubeflow/katib/pull/1758#issuecomment-988151233
  - https://github.com/kubeflow/testing/pull/972